src/httpProblemDetailsDefinition.ts

```diff
@@ -0,0 +1,5 @@
+export interface HttpProblemDetailsDefinition {
+  type: string;
+  title: string;
+  status: number;
```

Comment:
This should be a valid HTTP status, not only any number; can we highlight it by the type here?